pkgs/by-name/im/imhex/package.nix

```diff
@@ -82,6 +82,7 @@ stdenv'.mkDerivation (finalAttrs: {
 
   nativeBuildInputs =
     [
+      makeWrapper
```

Comment:
Note that this makes the `lib.optionals stdenv.isDarwin [ makeWrapper ]` line below redundant.(=> See https://github.com/NixOS/nixpkgs/pull/355911)